单目摄像头加IMU调研.md

```
@@ -0,0 +1,23 @@
+# 单目摄像头加IMU调研
+| 方法                           | 类型                               | 优点                                            | 缺点                                             | 工程建议                      |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)